docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -193,12 +203,16 @@ When passing arguments, follow these rules:
 
 #### kotlin.daemon.jvmargs property
 
-You can add the `kotlin.daemon.jvmargs` property in the `gradle.properties` file:
+You can add the `kotlin.daemon.jvmargs` property in the `gradle.properties` file. Note that if you do not specify `ReservedCodeCacheSize` here or in Gradle's JVM args, a default will be applied by the Kotlin Gradle plugin:
 
 ```none
 kotlin.daemon.jvmargs=-Xmx1500m -Xms500m
 ```
 
+```none
```

Comment:
```suggestion
Note that if you don't specify the `ReservedCodeCacheSize` argument here or in Gradle's JVM arguments, the Kotlin Gradle plugin applies a default value of `320m`:

```none
```
I think that it's better to have this sentence and it's sample separate from the first. Otherwise the second sample looks lost.